plugin/src/test/sc/plugin2021/xstream/ConverterTest.kt

```diff
@@ -155,6 +148,13 @@ class ConverterTest: WordSpec({
                       <second displayName="">
                         <color class="team">TWO</color>
                       </second>
+                      <board>
+                        <field x="0" y="0" content="RED"/>
+                        <field x="1" y="3" content="GREEN"/>
+                        <field x="8" y="6" content="YELLOW"/>
+                        <field x="5" y="9" content="BLUE"/>
+                      </board>
+                      <lastMoveMono class="linked-hash-map"/>
```

Comment:
is it fine to just reorder the XML?

Reply:
Depends on how the pupils parse it 😕 I would prefer to not change it mid-season.